app/src/main/res/values-xmf/strings.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Authors:
+* Narazeni
+-->
+<resources>
+  <string name="file_system_does_not_support_4gb">თქვანი ფაილიშ სისტემა ვაუხუჯანს 4გბ-შე უმოს ფაილეფს</string>
```

Comment:
I am comparing the translation of the `Georgian` language to `English`, and I have noticed that the translation does not match the content in the `string.xml` file. The translations in this file appear different compared to the English version. However, I have observed that the translations in the other files are similar to our string file.
![Screenshot from 2023-07-19 18-01-38](https://github.com/kiwix/kiwix-android/assets/34593983/ddef49c7-ab2d-4ff4-b0be-b6a5a9ad3a76)
 

![Screenshot from 2023-07-19 18-01-49](https://github.com/kiwix/kiwix-android/assets/34593983/41bd9f93-4d34-48b3-a210-829fb7da7937)

Reply:
Please report this upstream, on TW, and lets wait a bit to see if this can be corrected.

Reply:
@kelson42, I have made the changes on upstream and those changes are come in the latest commit of the translate wiki.